src/model_search.jl

```diff
@@ -137,7 +137,17 @@ function MLJBase.models(conditions...)
     return sort!(unsorted)
 end
 
-MLJBase.models() = models(x->true)
```

Comment:
We don't need this `method` actually, since `all(c for c in ())` will return `true` anyway.